src/common/meta/src/key/schema_name.rs

```diff
@@ -220,6 +220,17 @@ impl SchemaManager {
         Ok(())
     }
 
+    /// Updates a [SchemaNameKey].
```

Comment:
Let's pass the `DeserializedValueWithBytes<SchemaNameValue>` instead of `SchemaNameValue` into method.

```suggestion
    pub async fn get_raw(
        &self,
        schema: SchemaNameKey<'_>,
    ) -> Result<Option<DeserializedValueWithBytes<SchemaNameValue>>> {
        let raw_key = schema.to_bytes();
        self.kv_backend
            .get(&raw_key)
            .await?
            .map(|x| DeserializedValueWithBytes::from_inner_slice(&x.value))
            .transpose()
    }
    
   /// Updates a [SchemaNameKey].
```

You need to implement the `MetadataValue` trait for it.
```rust
impl_metadata_value! {
  SchemaNameValue
}
```